File: C#/1008 Salário.cs
using System;

class MainClass {
  public static void Main (string[] args) 
  {
   int N = Convert.ToInt32(Console.ReadLine());   
   int V = Convert.ToInt32(Console.ReadLine());
   double H = Convert.ToDouble(Console.ReadLine());
   double SALARY;
   
   SALARY = H*V;
   Console.WriteLine("NUMBER = {0}",N);
   Console.WriteLine("SALARY = U$ {0:0.00}",SALARY);
  }
}

File: C++/1008 Salário.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 int a, b;
  double c, x;
  scanf("%d %d", &a, &b);
  scanf("%lf", &c);
  x = b*c;
  printf("NUMBER = %d\n", a);
  printf("SALARY = U$ %4.2lf\n", x);
  return 0;
}
